compiler-core/src/warning.rs

```diff
@@ -1092,30 +1105,30 @@ See: https://tour.gleam.run/functions/pipelines/",
                     };
 
                     Diagnostic {
-                        title: "Incompatible gleam version range".into(),
-                        text: wrap(&format!(
-                        "{feature} introduced in version v{minimum_required_version}. But the Gleam version range \
+                                        title: "Incompatible gleam version range".into(),
+                                        text: wrap(&format!(
+                                        "{feature} introduced in version v{minimum_required_version}. But the Gleam version range \
                         specified in your `gleam.toml` would allow this code to run on an earlier \
                         version like v{wrongfully_allowed_version}, resulting in compilation errors!",
-                    )),
-                        hint: Some(format!(
-                            "Remove the version constraint from your `gleam.toml` or update it to be:
+                                    )),
+                                        hint: Some(format!(
+                                            "Remove the version constraint from your `gleam.toml` or update it to be:
 
     gleam = \">= {minimum_required_version}\""
-                        )),
-                        level: diagnostic::Level::Warning,
-                        location: Some(Location {
-                            label: diagnostic::Label {
-                                text: Some(format!(
-                                    "This requires a Gleam version >= {minimum_required_version}"
-                                )),
-                                span: *location,
-                            },
-                            path: path.clone(),
-                            src: src.clone(),
-                            extra_labels: vec![],
-                        }),
-                    }
+                                        )),
+                                        level: diagnostic::Level::Warning,
+                                        location: Some(Location {
+                                            label: diagnostic::Label {
+                                                text: Some(format!(
+                                                    "This requires a Gleam version >= {minimum_required_version}"
+                                                )),
+                                                span: *location,
+                                            },
+                                            path: path.clone(),
+                                            src: src.clone(),
+                                            extra_labels: vec![],
+                                        }),
+                                    }
```

Comment:
Could you undo these unrelated changes please 🙏

Reply:
oh yeah, whoops, looks like my formatter causing trouble

Reply:
fixed